Zend/Zend.m4

```diff
@@ -256,7 +256,8 @@ AC_ARG_ENABLE([zend-signals],
   [AS_HELP_STRING([--disable-zend-signals],
```

Comment:
It would be nice to adapt that regarding ZTS (i.e. display `--enable-zend-signals` there).

Reply:
How could we do that? AFAIK `AS_HELP_STRING` is only used to generate the help message, at this point we don't know if ZTS will be enabled or not.

Reply:
Oh, indeed. Never mind. :)